src/TextField/TextField.tsx

```diff
@@ -34,16 +37,20 @@ export default function TextField({
   // return components
   return (
     <MuiTextField
+      data-testid="text-field"
       disabled={disabled}
       error={error}
       fullWidth
       helperText={helperText}
       label={label}
       margin={margin}
+      multiline={multiline}
       onChange={onChange}
       placeholder={placeholder}
       InputProps={!isFieldMasked ? undefined : InputProps}
       required={required}
+      minRows={minRows}
```

Comment:
minRows and maxRows are only valid when multiline is set to true. Make sure you only set min/max rows when multiline is actually true.